app/assets/javascripts/utils/eventbus.js

```diff
@@ -0,0 +1,3 @@
+import Vue from 'vue';
+
+export default new Vue({});
```

Comment:
It returns a plain new Vue object with built-in methods like `$emit`, `$on`, `$off`. It's the same thing as doing `$({})` in jQuery like I tried to do [here](https://github.com/kubic-project/velum/pull/167/files#diff-3a404e107e5ad941e98771e088f6cbc1).

I did this to simulate a pub/sub object and let different components communicate without compromising everything with coupling. As an example, normally the child/inner components shouldn't handle data/services requests as a good practice. It should bubble an event and let the page/root parent component handle all of this. This way if we need to check how data is being handled at first, we will know where to take a look at. 

Not sure if I explained well enough for you. Let me know. :)